### ISSUE TITLE
[ML_modeling] 임세훈

### ISSUE THREAD

**Issue by mnxcv** [HanyangTechAI/2023-Summer-Project#13]
### ML_Modeling
kaggle 닉네임: mnxcv1
![image](https://github.com/HanyangTechAI/2023-Summer-Project/assets/100228015/52fa9c74-20c3-4d0e-8078-65c04fad2025)

```py
args = easydict.EasyDict({
  "train_path" : "./data/train.csv",
  "valid_path" : "./data/valid.csv",
  "device" : 'cpu',
  "mode" : "train",
  "batch" : 32,
  "maxlen" : 128,
  "lr" : 2e-5,
  "eps" : 1e-8,
  "epochs" : 3,
  "model_ckpt" : "monologg/koelectra-base-v3-generator",
})

if torch.cuda.is_available():
    args.device = 'cuda'
```
#### 기본 Baseline 코드에서 Batch, learning rate, epochs, model을 변경하였습니다.
- batch size와 learning rate는 Data의 크기와 Default값을 고려하여 줄이는 방향으로 수정했습니다.
- model은 기존 model의 unk토큰 문제를 해결하기 위해 Tokenizing 후의 배열을 찍는 코드를 실행해보며 적절한 model을 찾았습니다.
- ELECTRA Model이 자원을 적당히 사용하며 성능이 좋았기 때문에 사용했습니다.
- 코드를 실행해본 결과 epoch는 크게 중요하지는 않았고, 오히려 너무 크면 overfitting 문제가 발생하는 것을 확인해서 적당히 작게 설정했습니다.
